apps/android/LiveVideoApp/.gitignore

```diff
@@ -0,0 +1,65 @@
+*.iml
```

Comment:
I did not think to have multiple .gitignore files but I guess it makes quite a lot of sense for a monorepo.